### PR TITLE
added comment for Mongo requirements in values.yaml

### PR DESCRIPTION
**Pull request by Infinityxx**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by eldada** [comment]
@Infinityxx - good comment (and an important one!).
Can you also bump change version and add an entry to `CHANGELOG.md`?

File: stable/xray/values.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Default values for Xray HA.
+#Default values for Xray HA.
```
Comment (eldada) [inline stable/xray/values.yaml]: Return space after `#`. Will fail lint.

**Comment by eldada** [comment]
Need to resolve conflicts. Might be using an outdated version.

File: stable/xray/CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,11 @@
 # JFrog Xray Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [0.10.1] - March 10, 2019
+* Updated values.yaml added an important comment for the MongoDB requirements.
+
 ## [0.10.0] - Mar 3, 2019
-* Support loggers sidecars to tail a configured log
+*  Support loggers sidecars to tail a configured log
```
Comment (eldada) [inline stable/xray/CHANGELOG.md]: Remove extra space here.

**Comment by jainishshah17** [comment]
@eldada @rimusz Any Idea why CI is failing?

**Pull request merged**